Code/Yospace/README.md

```diff
@@ -0,0 +1,96 @@
+   2. If it is still not working, make sure to remove any `THEOplayerSDK.xcframework` inclusions that you manually installed before installing this connector package.
+
+### [Cocoapods](https://guides.cocoapods.org/using/getting-started.html#getting-started)
+
+1. Create a Podfile if you don't already have one. From the root of your project directory, run the following command: `pod init`
+2. To your Podfile, add the Yospace connector pods that you want to use in your app: `pod 'THEOplayer-Connector-Yospace'`
+3. Install the pods using `pod install` , then open your `.xcworkspace` file to see the project in Xcode.
+
+To support custom feature builds of THEOplayerSDK perform the following steps:
+
+1. Clone this repository to your computer.
+2. Use a [local override](https://guides.cocoapods.org/using/the-podfile.html#using-the-files-from-a-folder-local-to-the-machine) of the `THEOplayerSDK-basic` pod by adding the following line to your projects Podfile: `pod 'THEOplayerSDK-basic', :path => 'iOS-Connector/Helpers/TheoPod'` and make sure the path points to the [TheoPod folder](../../Helpers/TheoPod).
+
+## Dependencies
+
+The THEOPlayer Yospace connector has two dependency frameworks: THEOplayerSDK and YOAdManagement.
+
+THEOplayerSDK is added as a dependency on both Cocoapods and SPM and will be fetched by each dependency manager.
+YOAdManagement is published as a private pod hosted on Artifactory by jfrog. In order to get the framework you will need to:
+1. Setup a Yospace developer account at https://www.yospace.com/developer
+2. Login to the Yospace Apple docs https://developer.yospace.com/sdk-documentation/apple/userguide/latest/en/index-en.html
+3. Follow the instructions to setup the artifactory, authenticate, and fetch the framework at https://developer.yospace.com/sdk-documentation/apple/userguide/latest/en/prerequisites.html
+
+## Usage
+
+Import the `THEOplayerConnectorYospace` module
+
+```swift
+import THEOplayerConnectorYospace
+```
+
+Create a `SourceDescription` that defines the Yospace stream:
+
+```swift
+let streamType: YospaceStreamType = .live
+let typedSource: TypedSource = .init(
+    src: "yospace_live_stream_url",
+    type: "application/x-mpegurl",
+    ssai: YospaceServerSideAdInsertionConfiguration(streamType: streamType)
+)
+let source: SourceDescription = SourceDescription(source: typedSource)
+```
+
+Create a `YospaceConenctor` that uses a `THEOplayer` instance:
+
+```swift
+let theoplayer: THEOplayer = .init()
+let connector: YospaceConnector = .init(player: theoplayer)
+```
+
+**Note:** Hold a reference to your connector instance by storing it as a property. Keeping it inline will get it auto-released from memory. Once the connector is released from memory it will reset and unload autonomously.
+
+Set the source and play. There are 2 ways to set a Yospace source:
+
+1. Through the API provided by the YospaceConnector:
+
+```swift
+connector.setupYospaceSession(sourceDescription: source)
```

Comment:
maybe it's time to retire this API?

Reply:
We can. Web still has it so I added it here too.

Reply:
We could put `player.source` first and `setupYospaceSession()` second here, to guide customers towards the newer API. 😉